### PR TITLE
Plo ord issue front end

### PR DESCRIPTION
**Pull request by lewisrenfrew**

### CODE REVIEW

File: src/Facade/ResourceBuilders/AddressResourceBuilder.cs [hunk 1]
```diff
@@ -21,7 +21,7 @@ public AddressResource Build(Address address)
                 Line3 = address.Line3,
                 Line4 = address.Line4,
                 PostCode = address.PostCode,
-                Country = address.Country,
+                Country = address.Country.Name,
```
Comment (iaincrawf) [inline src/Facade/ResourceBuilders/AddressResourceBuilder.cs]: Some of the data is crap for addresses so I think we sometimes have null in the country column - not sure if we need just like `Country = address.Country?.name` or if we also need to make Country nullable but think we might need to check for it

File: src/Service.Host/client/src/actions/index.js [hunk 2]
```diff
@@ -297,6 +297,11 @@ export const failedPartsReportActionTypes = makeReportActionTypes(
     reportTypes.failedPartsReport.actionType
 );
 
+export const issueSernosActionTypes = makeProcessActionTypes(processTypes.issueSernos.actionType);
+
+export const buildSernosActionTypes = makeProcessActionTypes(processTypes.buildSernos.actionType);
+
+export const purchaseOrderActionTypes = makeActionTypes(itemTypes.purchaseOrder.actionType);
 export const daysRequiredReportActionTypes = makeReportActionTypes(
```
Comment (iaincrawf) [inline src/Service.Host/client/src/actions/index.js]: Will linting want an extra line in here?

File: src/Service.Host/client/src/reducers/index.js [hunk 3]
```diff
@@ -209,6 +212,16 @@ const reducer = history =>
         worksOrderLabel,
         worksOrderLabels,
         workStations,
+        componentCounts,
+        labelPrint,
+        labelPrinters,
+        labelPrintTypes,
+        suppliers,
+        addresses,
```
Comment (iaincrawf) [inline src/Service.Host/client/src/reducers/index.js]: If you can be bothered I'm sure Richard or someone had sorted these all alphabetically but it obviously isn't essential!

File: src/Service.Host/client/src/actions/index.js (hunk 2 above)
Comment (lewisrenfrew) [reply src/Service.Host/client/src/actions/index.js]: no.. but consistency wants an extra line so will add it anyways

File: src/Facade/ResourceBuilders/AddressResourceBuilder.cs (hunk 1 above)
Comment (lewisrenfrew) [reply src/Facade/ResourceBuilders/AddressResourceBuilder.cs]: I think entity framework will be able to resolve the one to many relationship between countries and addresses ok like this

File: src/Domain.LinnApps/Country.cs [hunk 4]
```diff
@@ -7,5 +7,7 @@ public class Country
         public string CountryCode { get; set; }
 
         public string Name { get; set; }
+
+        public IEnumerable<Address> Addresses { get; set; }
```
[the remark this answers is not in the document]
Comment (lewisrenfrew) [reply src/Domain.LinnApps/Country.cs]: I think entity framework will be able to resolve the one to many relationship between countries and addresses ok like this

**Pull request merged**